packages/gatsby-source-drupal/src/gatsby-node.js

```diff
@@ -203,14 +205,22 @@ exports.sourceNodes = async (
   // Default skipFileDownloads to false.
   skipFileDownloads = skipFileDownloads || false
 
+  // Default language to `en`
+  language = language || `en`
+
+  // Default defaultLanguage to `en`
+  defaultLanguage = defaultLanguage || `en`
+
+  const languagePrefix = defaultLanguage !== language ? language : ``
```

Comment:
Probably better to just pass in `languagePrefix`. I think that makes the most sense from a DevX perspective. "I am trying to use an endpoint with a /es/ prefix, I put languagePrefix: es into the gatsby-config"